### ISSUE TITLE
Βήμα 5ο

### ISSUE THREAD

**Issue by gmagaf** [slp-ntua/patrec-labs#79]
Για την υλοποίηση του LSTM βασιστήκαμε στην δουλειά που είχαμε κάνει στο 2ο εργαστήριο, στο οποίο είχε λειτουργήσει αρκετά καλά, και αντιμετωπίζουμε σε αυτό το dataset το πρόβλημα το δίκτυο να μην μαθαίνει. Επομένως ακολουθήσαμε την υπόδειξη να προσπαθήσουμε να το οδηγήσουμε σε overfit σε μία παρτίδα 10-20 δειγμάτων και πάλι το ίδιο πρόβλημα, το accuracy μένει σταθερό κάτω από το 3% (στο train set). Δοκιμάσαμε επίσης να τυπώσουμε τα αποτελέσματα που μας δίνει το μοντέλο στο validation set και φαίνεται να υπερισχύει μία συγκεκριμένη κλάση που δίνει πάντοτε μεγαλύτερη τιμή.
Επομένως, επειδή το ψάχνουμε ήδη μερικές μέρες χρειαζόμαστε κάποιον τρόπο να προχωρήσουμε. Υπάρχει κάτι άλλο που μπορούμε να δοκιμάσουμε σε debugging; Κάποια ιδιαιτερότητα ίσως των ακολουθιών αυτών δημιουργεί δυσκολίες;

Ευχαριστώ εκ των προτέρων

**Comment by SergeantChris** [comment]
Δοκίμασε μικρότερο learning rate στον optimizer (π.χ. adam με 1e-4), σε εμένα αυτό έφταιγε (ταυτόχρονα σιγούρεψε ότι έχετε επαρκή αριθμό παραμέτρων hidden, πχ > 200).
